test/test_atacac/test_backup.py

```diff
@@ -0,0 +1,134 @@
+import os
+import shutil
+import textwrap
+from unittest import mock
+from test.utils import MockDir
+
+import yaml
+import pytest  # pylint: disable=import-error
+from click.testing import CliRunner
+
+from atacac import backup
+
+
+class TestDumper:
+
+    def test_increase_indent(self):
```

Comment:
Why this test is in class while all other tests are functions?

Reply:
I kept the convention:
* Class::method() <=> TestClass::test_method()
* function() <=> test_function()

Reply:
Should we stick to a better one?

Reply:
IMHO weird convention :confused: I prefer consistency

Reply:
What is your suggestion for [`Example::_is_valid()`](https://github.com/europ/atacac/blob/master/atacac/validators/builtin.py#L7) and [`Dict::_is_valid()`](https://github.com/europ/atacac/blob/master/atacac/validators/builtin.py#L14)?

Currently, it is named as `TestExample::test__is_valid()` and `TestDict::test__is_valid()` while it is in a test class.

Can it be a function such as `test_Example_is_valid()` and `test_Dict_is_valid()` ... following pattern of `test_<class>_<method>` ?

Reply:
Name doesn't really matter if it is descriptive. What about `test_example_validation`, `test_dict_validation`, `test_yaml_dumper_indentation`?

Reply:
Alright. I will add it.